apps/desktop/src/environments/environment.ts

```diff
@@ -12,7 +12,7 @@ export const environment = {
       CLIENT_ID: 'FAKE',
     },
     gdrive: {
-      endpoint: 'http://127.0.0.1:5000/gdrive/authorize',
+      endpoint: 'https://stashify.cloud/api/external_auth/gdrive/authorize',
```

Comment:
I saw you have a file replacement on dev mode, shouldn't this be in `environment.dev.ts` then?